CMakeLists.txt

```
@@ -35,8 +35,9 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-duplicate-decl-specifier \
 cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 
+set(CHIPSTAR_VERSION 1.1)
```

Let's bump this up so applications may tell release 1.1 and main branch (aka. the next release) apart.
```suggestion
set(CHIPSTAR_VERSION 1.2)
```